lib/grac/client.rb

```diff
@@ -172,16 +175,14 @@ def postprocessing(data, processing = nil)
       return data if @options[:postprocessing].nil? || @options[:postprocessing].empty?
 
       if data.kind_of?(Hash)
-        data.each do |key, value|
+        data.keys.reverse.each do |key|
```

Comment:
Could you explain the necessity of `.reverse` here?

On that note: Why did you move to looping over the keys instead of keys and values?